File: Calculator.py
# Simple calculator (Made: Jauaries).

import math

def add(x, y):
    a = x + y
    return a

def subtract(x, y):
    s = x - y
    return s

def multiply(x, y):
    m = x * y
    return m

def divide(x, y):
    d = x / y
    return d

def power(x,y):
    p = pow(x,y)
    return p


print("Select the operation that you want:")
print("1. Add")
print("2. Subtract")
print("3. Multiply")
print("4. Divide")
print("5. Power")

choice = input("Enter your choice(| 1 | 2 | 3 | 4 | 5 |): ")

num1 = float(input("Enter the first number: "))
num2 = float(input("Enter the second number: "))

if choice == '1':
    print("%s + %s = %s" %(num1, num2, add(num1, num2)))
elif choice == '2':
    print("%s - %s = %s" %(num1, num2, subtract(num1, num2)))
elif choice == '3':
    print("%s * %s = %s" %(num1, num2, multiply(num1, num2)))
elif choice == '4':
    print("%s / %s = %s" %(num1, num2, divide(num1, num2)))
elif choice == '5':
    print("%s ^ %s = %s" %(num1, num2, power(num1, num2)))
else:
    print("Invalid input.")

File: Calculator.java
//Simple calculator program coded in java (Made: Jauaries).

import java.util.Scanner;
import java.lang.Math;

public class Calculator {
    public static void main(String[] args) {
        double num1, num2;

        Scanner scanner = new Scanner(System.in);


        //Defining the numbers (num1 & num2)
        System.out.print("Enter first number: ");
        num1 = scanner.nextDouble();
        System.out.print("Enter second number: ");
        num2 = scanner.nextDouble();


        //Operators
        System.out.print("Enter an operator (+, -, *, /, ^): ");
        char operator = scanner.next().charAt(0);

        scanner.close();
        double output;


        //Defining the operators used in the program
        switch(operator) {
            case '+':
                output = num1 + num2;
                break;

            case '-':
                output = num1 - num2;
                break;

            case '*':
                output = num1 * num2;
                break;

            case '/':
                output = num1 / num2;
                break;

            case '^':
                output = Math.pow(num1, num2);
                break;


            //Error message if the user selects different operator, that isn't in the list
            default:
                System.out.print("ERROR: You have entered wrong operator");
                return;
        }

        System.out.println(num1 + " " + operator + " " + num2 + " = " + output);
    }
}
